addons/account/static/src/components/systray_item/systray_item.js

```diff
@@ -0,0 +1,32 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { Component, onWillStart } from "@odoo/owl";
+
+export class AccountSystrayItem extends Component {
+    setup() {
+        this.currentCompany = useService("company").currentCompany;
+        this.orm = useService("orm");
+
+        onWillStart(async () => {
+            this.display = await this.orm.call("res.company", "is_quick_edit_mode_enabled",[''],{},);
```

Comment:
It would be nice if there was a way to avoid doing this additional network call every time someone opens a tab, maybe by setting something in the context or the environment.

Reply:
I have used the 'session' to avoid making a network call.  In my understanding of the Odoo documentation, it is the way to manage to avoid the network call. Is it the correct way to do it ?